src/components/NetworkConnectivity.js

```diff
@@ -81,11 +81,11 @@ class NetworkConnectivity extends React.PureComponent<Props, State> {
     const { pingInterval } = this.props;
     const handler = this.getConnectionChangeHandler();
 
-    NetInfo.isConnected.addEventListener('connectionChange', handler);
+    NetInfo.addEventListener('connectionChange', handler);
```

Comment:
`addEventListener` only receives one argument, the handler, and returns an unsubscribe function. I don't see `removeEventListener` documented anywhere. Maybe it still works, but it should be a deprecated method.